src/main/resources/application.yml

```diff
@@ -31,6 +31,7 @@ documentRender:
 env:
   name: ${ENV_NAME}
   chsUrl: ${CHS_URL}
+  chsFinanceEmail: ${CHS_FINANCE_EMAIL}
```

Comment:
I wouldn't put this under the `env` key. I don't think the finance team's email relates to the environment config

Reply:
Which key would you recommend?

Reply:
Maybe this?

```
email:
  chsFinanceEmailAddress:
```